.github/workflows/check+deploy.yaml

```diff
@@ -20,6 +20,18 @@ jobs:
                 submodules: true
                 path: _src/
 
+            - name: Verify release version matches source code version
+              if: github.event == 'release' && startsWith(github.ref, 'refs/tags/v')
+              shell: bash
+              run: |
+                pushd _src
+                export TAG_VERSION=${GITHUB_REF##refs/tags/v}
+                export SRC_VERSION=$(sed -ne "s/^__version__\s*=\s*'\([^']*\)'/\1/p" wilson/_version.py)
```

Comment:
I would have gone with `SRC_VERSION=$(python3 -c "from wilson._version import __version__; print(__version__)")` but this seems to work ;)

Reply:
(might also be `python` in newer ubuntu)

Reply:
Ah, excellent. That's much more reliable. I had tried something similar with ``.wilson._version`` rather than ``wilson._version``, which failed. However, setting ``PYTHONPATH`` to ``./:$PYTONPATH`` should ensure that we use the source version, right?

Reply:
According to https://docs.python.org/3/tutorial/modules.html#the-module-search-path the current directory takes preference anyway, so we should be fine without adapting `PYTHONPATH`

Reply:
Understood, thanks!

Reply:
Sorry this took so long, Fixed, and will force push here in a bit.